core/deployment/src/main/java/io/quarkus/runner/RuntimeClassLoader.java

```diff
@@ -273,6 +277,10 @@ public InputStream getResourceAsStream(String nm) {
         }
     }
 
+    private String getPackageNameFromClassName(String className) {
+        return className.substring(0, className.lastIndexOf('.'));
```

Comment:
I think this won't account well for the default package case, since `lastIndexOf` will return -1.

Reply:
Indeed. I have updated the code to at least no fail at this stage when the default package is used. Note however that client code will fail if it tries to use the (null) package of a class in the default package - but of course we can't do anything about that :)